desktop/core/src/desktop/management/commands/rungunicornserver.py

```diff
@@ -76,6 +77,13 @@ def post_fork(server, worker):
   with open(PID_FILE, "a") as f:
     f.write("%s\n"%worker.pid)
 
+def post_worker_init(worker):
+  connection.connect()
+  atexit.unregister(_exit_function)
```

Comment:
What is the purpose of `atexit.unregister(_exit_function)` here?

Reply:
discussed this with @wing2fly and removed the atexit as this is not needed.